### PR TITLE
Some tiny text tweaks ('and empty' -> 'an empty')

### PR DESCRIPTION
**Pull request by jean**
Also 'class that groups' (singular verb to match subject),
and don't break commands across lines if we can avoid it.

### CODE REVIEW

**Comment by mathias-lang-sociomantic** [comment]
Looks like you need a rebase

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,9 @@
 
 prefix ?= /usr/local
+# fall back to $prefix/etc (like autoconf); see discussion on #222
+#sysconfdir ?= $(prefix)/etc)
```
Comment (nemanja-boric-sociomantic) [inline Makefile]: Is this supposed to be commented-out?

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,5 @@
 
 prefix ?= /usr/local
-# fall back to $prefix/etc (like autoconf); see discussion on #222
-#sysconfdir ?= $(prefix)/etc)
```
Comment (nemanja-boric-sociomantic) [inline Makefile]: Ah, ok, now I see :-)

**Comment by leandro-lucarella-sociomantic** [comment]
Waiting on the rebase :innocent:

**Comment by jean** [comment]
Finally ..

**Review by leandro-lucarella-sociomantic — APPROVED** [review]

**Comment by leandro-lucarella-sociomantic** [comment]
Thanks!

**Pull request merged**